src/components/documentBuilder/edit/getElementEditSchemas.tsx

```diff
@@ -167,33 +167,16 @@ function getElementEditSchemas(
         name: joinName(elementName, "config", "variant"),
         schema: {
           type: "string",
-          enum: [
-            "primary",
-            "secondary",
-            "success",
-            "warning",
-            "danger",
-            "info",
-            "light",
-            "dark",
-            "link",
-            "outline-primary",
-            "outline-secondary",
-            "outline-success",
-            "outline-warning",
-            "outline-danger",
-            "outline-info",
-            "outline-light",
-            "outline-dark",
-            "outline-link",
-          ],
+          format: "bootstrap-button-variant",
         },
+        uiSchema: { isClearable: false, isSearchable: false },
         label: "Button Style",
         description: "The style/variant of the button",
       };
       const sizeEdit: SchemaFieldProps = {
         name: joinName(elementName, "config", "size"),
         schema: { type: "string", enum: ["lg", "md", "sm"] },
+        uiSchema: { isClearable: false, isSearchable: false },
```

Comment:
This is OK, but if we want to be similar to how RJSF does their uiSchema, I think we may want to nest them under `props`?  See how RJSF does theirs: https://rjsf-team.github.io/react-jsonschema-form/docs/api-reference/uiSchema/

Reply:
It looks like we want to use `props` if we intend to pass the values through directly, which we are in this case.

I'll update this to 
```suggestion
        uiSchema: {
          options: {
            props:{
              isClearable: false, isSearchable: false
            }
          }
        },
```